packages/php-wasm/compile/build-assets/phpwasm-emscripten-library.js

```diff
@@ -401,10 +401,21 @@ const LibraryExample = {
 	},
 
 	js_module_onMessage: function (data) {
+		if (typeof Asyncify === 'undefined') {
```

Comment:
Ok, debugged it as far as here: Asyncify is not defined for web.

Reply:
Ok, it works when I set `WITH_WS_NETWORKING_PROXY` to `yes` for web. Not sure why Asyncify is otherwise not needed for web.

Reply:
Without the WS proxy, Asyncify is needed just to support this PR. That's fine, though. But let's remove this `if` to immediately surface any errors related to Asyncify being undefined.